promptsource/templates/GEM/totto/templates.yaml

```diff
@@ -0,0 +1,166 @@
+dataset: GEM/totto
+templates:
+  281a24d1-d4ef-46c7-8759-f3f3168426ab: !Template
+    answer_choices: null
+    id: 281a24d1-d4ef-46c7-8759-f3f3168426ab
+    jinja: "{% if table_page_title != \"\" %}\nGiven the following details:\nTable\
+      \ Section Title: {{table_section_title}},\n{% if table_section_text != \"\"\
+      \ %}\nTable Section Text: {{table_section_text}},\n{% endif %}\nand the following\
+      \ original text describing the highlighted cell of the table: \n{{sentence_annotations[0][\"\
+      original_sentence\"]}},\nCan you guess the table's page title?\n|||\n{{table_page_title}}\n\
+      {% endif %}\n\n"
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: false
+    name: guess the table page title
+    reference: ''
+  2f1c60e2-6cd6-44c3-a722-8ccc185bce34: !Template
+    answer_choices: null
+    id: 2f1c60e2-6cd6-44c3-a722-8ccc185bce34
+    jinja: 'Given the following details:
+
+      {% if table_webpage_url != "" %}
+
+      Table Page Title: {{table_page_title}},
+
+      Table Section Title: {{table_section_title}},
+
+      {% if table_section_text != "" %}
+
+      Table Section Text: {{table_section_text}},
+
+      {% endif %}
+
+      and the following original text describing the highlighted cell of the table:
+
+      {{sentence_annotations[0]["original_sentence"]}},
+
+      Guess the URL from which this table has been extracted!
+
+      |||
+
+      {{table_webpage_url}}
+
+      {% endif %}
+
+
+      '
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: false
+    name: guess the table webpage url
+    reference: ''
+  7a2fb34a-c792-41b3-9682-fcda15f1d100: !Template
+    answer_choices: null
+    id: 7a2fb34a-c792-41b3-9682-fcda15f1d100
+    jinja: '{% if table_section_title != "" %}
+
+      Given the following details:
+
+      Table Page Title: {{table_page_title}},
+
+      {% if table_section_text != "" %}
+
+      Table Section Text: {{table_section_text}},
+
+      {% endif %},
+
+      and the following original text describing the highlighted cell of the table:
+
+      {{sentence_annotations[0]["original_sentence"]}},
+
+      Can you guess the table''s section title?
```

Comment:
Rereading the original paper I'm not sure correctly describes the original task? It appears that the original task was descriptions of table given the table as formatted?